ompi/mpi/fortran/mpif-h/register_datarep_f.c

```diff
@@ -89,28 +90,28 @@ static int extent_intercept_fn(MPI_Datatype type_c, MPI_Aint *file_extent,
 /* Data structure passed to the intercepts (see below).  It is an OPAL
    list_item_t so that we can clean this memory up during
    MPI_FINALIZE.  */
-typedef struct intercept_extra_state {
+typedef struct ompi_intercept_extra_state {
```

Comment:
I'm a little surprised that this type name would be exposed externally.  Are we using it outside of this file?